pkg/state/models.go

```diff
@@ -90,8 +91,17 @@ type V1 struct {
 	Certs map[string]util.CertType `json:"certs,omitempty" yaml:"certs,omitempty" hcl:"certs,omitempty"`
 }
 
+type License struct {
```

Comment:
This struct is duplicated, intentional? It's in pkg/state and pkg/api.

Reply:
I'll double check.. I was following the existing pattern.